lib/plugins/cob.js

```diff
@@ -310,6 +310,25 @@ function init(ctx) {
       , intentHandler: alexaCOBHandler
     }]
   };
+  
+   function googleHomeCOBHandler(result, next, sbx) {
+    var preamble = result && result.parameters && result.parameters.givenName ? result.parameters.givenName + ' has' : 'You have';
+    var value = 'no';
+    if (sbx.properties.cob && sbx.properties.cob.cob !== 0) {
+      value = Math.round(sbx.properties.cob.cob);
+    }
+    var response = preamble + ' ' + value + ' carbohydrates on board';
+    next(response);
+  }
```

Comment:
I wonder how we could change this code (and the Alexa implementation) so that COB plugin would really only have the COB features and any other plugin that needs COB data would contain the COB processing in the plugin itself? I actually failed the Alexa code review - we should aim for clean separation of concerns and cob.js containing any code that has to do with Alexa and Home is architecturally nasty.